### PR TITLE
Release 0.10

### PR DESCRIPTION
**Pull request by azriel91**
Cross linking: https://github.com/amethyst/documentation/issues/6.

### CODE REVIEW

File: src/content/blog/release-0.10.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+This change paves the way towards using the various [rustsim] libraries.
+This is a collection of high-quality libraries, developed and maintained by the broader Rust
+community to perform simulations.
+
+Going forward we will be working on integrating these libraries.
+In the meantime, [`amethyst-rhusics`] has been
+updated to support `nalgebra`.
+
+This is of course a breaking changes. To help our users in this transition we've written a
```
Comment (LucioFranco) [inline src/content/blog/release-0.10.md]: I think this should be singular, `This is of course a breaking change.`

File: src/content/blog/release-0.10.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+[testing]: https://www.amethyst.rs/book/latest/testing.html
+[api]: https://www.amethyst.rs/doc/latest/doc/amethyst_test/index.html
+
+### Other Changes
+
+Amethyst 0.10.0 involves a significant number of changes.
+We can't detail all of them here, but following are some of the more notable ones:
+
+* Out canonical Pong tutorial tutorial has been improved and expanded.
```
Comment (LucioFranco) [inline src/content/blog/release-0.10.md]: Our?

File: src/content/blog/release-0.10.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+* We can now draw images without having to define a `SpriteSheet` ([#1153]).
```
Comment (LucioFranco) [inline src/content/blog/release-0.10.md]: I think we should move away from using `We` and more towards `Images can now be drawn without having to define a SpriteSheet`.

File: src/content/blog/release-0.10.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+* `amethyst_ui::get_default_font` now supports loading fonts from the filesystem ([#1108]).
+* It is now easier than ever to build simple prefabs using `#[derive(PrefabData)]` ([#1035]).
+* We’ve been working on laminar for a while and we’re happy to announce that we
```
Comment (LucioFranco) [inline src/content/blog/release-0.10.md]: I would rephrase this as `The networking team has been working hard on laminar and it has finally been released...`

File: src/content/blog/release-0.10.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+* The web team is working on improving https://amethyst.rs.
+* The assets team is working on a new strategy for defining and loading assets based on
+  modern best-practices.
+* The rendering team is working hard on moving from our current OpenGL-based renderer, to one that
+  uses [gfx-rs]. This will empower Amethyst to start making use some of the goodies that comes with
+  modern graphics APIs.
+* Our community team is currently improving our contribution guidelines, and are trying to drive
+  and visualize our donation targets.
+* The networking team is working on their roadmap. If you are interested, have questions, or you
+  just want to be up to date with our work, we recommend that you check out the
```
Comment (LucioFranco) [inline src/content/blog/release-0.10.md]: should be changed too `with their work`

**Review by LucioFranco — COMMENTED** [review]
Looking good! Thanks for doing this. 👍

**Comment by azriel91** [comment]
@LucioFranco Thanks for the review, should be all fixed now.
@fhaynes Does the footnote match what you had in mind?

**Comment by fhaynes** [comment]
Yup, that is great, thank you!

**Review by LucioFranco — APPROVED** [review]
👍 thanks for doing this.

File: src/content/blog/release-0.10.md [hunk 6]
```diff
@@ -0,0 +1,212 @@
...
+
+[testing]: https://www.amethyst.rs/book/latest/testing.html
+[api]: https://www.amethyst.rs/doc/latest/doc/amethyst_test/index.html
+
+### Other Changes
+
+Amethyst 0.10.0 involves a significant number of changes.
+We can't detail all of them here, but following are some of the more notable ones:
+
+* Our canonical Pong tutorial tutorial has been improved and expanded.
```
Comment (torkleyy) [inline src/content/blog/release-0.10.md:98]: Maybe the state lifetime, too? It's a minor, but it's something that's been used as a reference for things that make Amethyst harder to understand, and it would be nice to use that as example here for work in this direction.

**Review by torkleyy — APPROVED** [review]
Lovely, thank you!

File: src/content/blog/release-0.10.md (hunk 6 above)
Comment (azriel91) [reply src/content/blog/release-0.10.md:98]: Yeap that's certainly notable 💯, added!

**Comment by LucioFranco** [comment]
@azriel91 whats the status on this?

**Comment by azriel91** [comment]
It's done, but waiting for the actual crates to be released before we publish (I think merging this automatically publishes, but I haven't checked CI config to confirm).

Oh ya, I'm out until late tonight, so won't be able to do the crate publishing

**Review by udoprog — APPROVED** [review]

**Comment by LucioFranco** [comment]
Happy v0.10.0 release everyone! 🎉
